RadioMark/index.js

```diff
@@ -0,0 +1,64 @@
+import React, { PropTypes } from 'react'
+import { Motion, spring } from 'react-motion'
+import compose from '../lib/compose'
+import themeable from '../decorators/themeable'
+
+const grayLines = '#cbcbcd'
+const klarnaBlue = '#0074c8'
+const white = 'white'
+
+function RadioMark ({ customize, checked, disabled, onClick, lowFPS }) {
+  const backgroundColor = customize ? customize.backgroundColor : klarnaBlue
+  const bulletColor = customize ? customize.bulletColor : white
+  const scale = checked ? 1 : 3
+
+  return (
+    <svg width='14' height='14' onClick={onClick}>
+      <Motion style={{ scale: lowFPS ? scale : spring(scale) }}>
+        {({ scale }) => (
+          <g>
+            <circle cx='7' cy='7' r='7' fill={grayLines} />
+            <circle cx='7' cy='7' r='7' fill={disabled ? grayLines : backgroundColor} style={{ opacity: 1 - (scale - 1) / 2 }} />
+            {/* tranformOrigin using matrices: http://stackoverflow.com/questions/6711610/how-to-set-transform-origin-in-svg#6714140 */}
+            <g transform={`matrix(${scale}, 0, 0, ${scale}, ${7 - scale * 7}, ${7 - scale * 7})`}>
```

Comment:
Got a lot of magic numbers here. What's the significance of 7? Is it the circle diameter? Then it should probably be extracted and referenced from the circle as well.